checkST2110.js

```diff
@@ -701,37 +736,43 @@ const test_20_72_2 = (sdp, params) => {
 
 const greatestCommonDivisor = (a, b) => !b ? a : greatestCommonDivisor(b, a % b);
 
+const checkExactframerate = (stream = {}) => {
+  let errors = [];
+  let frMatch = stream.exactframerate.match(frameRatePattern);
```

Comment:
I think you're right the `NaN` checks may be redundant after the pattern check, but I _think_ most of the other checks are useful (e.g. identifying `30/1`, `1/30` are wrong, as well as `60/2`), so I'm inclined to leave as is?

Reply:
Although I see we don't check specifically for denominator being zero before dividing by it, so that _is_ relying on the pattern ruling out that being zero!

Reply:
With regard to providence, this function has simply been factored out of  `test_20_72_3` so it could be reused in the new `test_40_2023_7_3`.  No test conditions were added or removed.

Reply:
OK I'll remove the redundant isNan checks